config/webpack.target.mobile.js

```diff
@@ -20,7 +20,6 @@ module.exports = function (production, app) {
         __ALLOW_HTTP__: !production,
         __TARGET__: JSON.stringify('mobile'),
         __SENTRY_TOKEN__: JSON.stringify('29bd1255b6d544a1b65435a634c9ff67'),
-        __DEVMODE__: !production,
```

Comment:
cannot see the point with sentry

Reply:
I'm not sure I understand what you mean here.

Reply:
You create a PR named `chore: cleaning up sentry logs` and I saw something not related.
I guess it is easier for you to put the whole work day in a simple Pull Request, but as I wrote earlier on #398 it is less mess to do PR related to one and only one consideration. Let's group our work by topic instead of `git commit -am 'my work for the day' &&  git push && git pull-request -m 'please sort my work for me and review'`.

Reply:
Ah I see, but I wouldn't say that's the case here :). If I didn't remove the `__DEVMODE__` flag in this PR, I would be using both `__DEVLOPMENT__` and `__DEVMODE__` inside the same file (`reporter.js`) — and that smells. So I did some light refactoring along the way, as we said we'd do :)

Reply:
Maybe its Friday and I lose all the patience of mine in #398

Reply:
No problem ;). I'm still looking for my courage before diving into it.